### PR TITLE
[PORT][9.0] product_m2m_categories/product_multi_category

### PR DESCRIPTION
**Pull request by SodexisTeam**

### CODE REVIEW

**Comment by SodexisTeam** [comment]
OCA/ocb travis fails due to the pending PR https://github.com/OCA/OCB/pull/454.

[Note: probably need to restart travis job once this PR is merged.]

File: product_multi_category/README.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (pedrobaeza) [inline product_multi_category/README.rst]: Please use latest template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

File: product_multi_category/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# © 2009 Sharoon Thomas, Guewen Baconnier, Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Product Multi Categories",
+    "version": "9.0.1.0.0",
+    "author": "Sharoon Thomas,Sodexis,Odoo Community Association (OCA)",
```
Comment (pedrobaeza) [inline product_multi_category/__openerp__.py]: You should put here the rest of the contributors: Akretion, Camptocamp and Avanzosc

File: product_multi_category/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    "contributors": ["Mikel Arregi <mikelarregi@avanzosc.es>"],
+    "website": "",
+    "license": "AGPL-3",
+    "category": "Product Management",
+    "depends": ['base', 'product'],
+    "data": [
+            'views/product_view.xml'
```
Comment (pedrobaeza) [inline product_multi_category/__openerp__.py]: Is this a tab or maybe more spaces than needed?

Comment (atchuthan) [inline product_multi_category/__openerp__.py]: 4 spaces

Comment (pedrobaeza) [inline product_multi_category/__openerp__.py]: Well, it seems to be more than 4 spaces or to have a tab in the middle.

**Comment by pedrobaeza** [comment]
I'm thinking that it can be interesting to use the many2many_tags widget and put in the same place as the partner tags instead using a full notebook page for it. What do you think?

**Comment by SodexisTeam** [comment]
@pedrobaeza, yes I would agree to use man2ymany_tags.
Can I have this field in Inventory page (below Internal category)?

**Comment by pedrobaeza** [comment]
Yeah, better to put there. I see that in v9, partner categories are also removed from the header and put after the address

**Comment by atchuthan** [comment]
@pedrobaeza changes done

File: product_multi_category/models/product.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+from openerp import models, fields
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    categ_id = fields.Many2one(string='Pricing/Primary Category')
+    categ_ids = fields.Many2many(
+        comodel_name='product.category', relation='product_categ_rel',
+        column1='product_id', column2='categ_id', string='Product Categories')
```
Comment (pedrobaeza) [inline product_multi_category/models/product.py]: Put `string="Extra categories"` for being easily recognized in the UI now that we don't have the tab.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by guewen** [comment]
:+1:

**Comment by atchuthan** [comment]
@guewen  thanks for the merge!!

**Comment by atchuthan** [comment]
@pedrobaeza I have added a PR for module name change at https://github.com/OCA/OpenUpgrade/pull/462

**Pull request merged**